registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/util/CbeffToBiometricUtil.java

```diff
@@ -295,7 +296,8 @@ public InputStream extractCbeffWithTypes(String cbeffFile, List<String> types) t
 	 *             the exception
 	 */
 	public List<BIR> getBIRTypeList(String cbeffFileString) throws Exception {
-		return cbeffutil.getBIRDataFromXML(CryptoUtil.decodeBase64(cbeffFileString));

```

Comment:
this should be fixed in the test case and not in the code.
For example - 
you will see below code to read cbeff file string. Just encode with base64 and send it. 
Existing code 
-----------------
InputStream inputStream = new FileInputStream(cbeffFile);
String cbeff = IOUtils.toString(inputStream, "UTF-8");

Change required
------------------
InputStream inputStream = new FileInputStream(cbeffFile);
String cbeff = CryptoUtil.encodeBase64(IOUtils.toString(inputStream, "UTF-8").getBytes());